tests/foreman/cli/test_repository.py

```diff
@@ -228,8 +229,14 @@ def test_positive_create_with_default_download_policy(self):
 
         @Assert: YUM repository with a default download policy
         """
+        default_dl_policy = entities.Setting().search(
```

Comment:
It's CLI test, please use hammer's `settings` command instead (i believe smth like `settings list --search=name=default_download_policy` should work)